package.json

```diff
@@ -21,26 +21,23 @@
 		"@byba/eslint-config": "^1.4.0",
 		"eslint": "^8.33.0",
 		"husky": "^8.0.3",
-		"lint-staged": "^13.1.0"
+		"lint-staged": "^13.1.0",
+		"chalk": "4",
+		"gulp": "^4.0.2",
+		"gulp-rename": "^2.0.0",
+		"prompt-sync": "^4.2.0"
 	},
 	"dependencies": {
 		"@byba/express-validator": "^1.1.3",
 		"body-parser": "^1.20.1",
-		"chalk": "4",
 		"connect-session-sequelize": "^7.1.5",
-		"del": "6",
 		"express": "^4.18.2",
 		"express-form-data": "^2.0.19",
 		"express-handlebars": "^6.0.7",
 		"express-session": "^1.17.3",
-		"gulp": "^4.0.2",
-		"gulp-rename": "^2.0.0",
-		"jimp": "^0.16.2",
 		"joi": "^17.7.0",
 		"morgan": "^1.10.0",
-		"prompt-sync": "^4.2.0",
-		"sequelize": "^6.28.0",
-		"serve-favicon": "^2.5.0"
```

Comment:
I don't know where this got lost but we definitely used to use this - and we should add it back in as it seems like it's missing from the current source

Reply:
They should be in dev dependencies now as I thought they were only used for gulp n that?

Reply:
Sorry, yes my comment was only on 'serve-favicon' but it looks like github highlighted everything

Reply:
Ah gotcha, thanks GitHub 👀

If I'm honest, do we actually need it? We don't currently have a favicon and I'm fairly sure we can show one with just html?